test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -688,6 +688,16 @@ ConstructorTestClass(int i, bool b)
         $res.CompletionMatches[0].CompletionText | Should -BeExactly '$MyDataVar'
     }
 
+    It 'Should complete global variable without scope' {
+        $res = TabExpansion2 -inputScript '$Global:MyTestVar = "Hello";$MyTestV'
+        $res.CompletionMatches[0].CompletionText | Should -BeExactly '$MyTestVar'
+    }
```

Comment:
Does still work with scope? Have we tests?

Reply:
Yes scopes still work. Line 1893 seems to have a test for a global scope variable specifically but there's also my other new test for "using" which may be in a different scope, but the logic remains the same.